src/actions/projectActions.js

```diff
@@ -1,33 +1,21 @@
 import * as types from "../constants/ActionTypes";
-import { scenes, storageRef } from "../firebase.js";
+
+const sceneRef = "/apiv1/scenes";
+const previewRef = "/apiv1/scenes/preview";
 
 export function asyncUserProj(id) {
     // fetch user's project
     return (dispatch) => {
         if (id) {
-            let userVals = [];
-            scenes.where("uid", "==", id).get().then(snap => {
-                let num_proj_loaded = 0;
-                snap.forEach(doc => {
-                    storageRef.child(`/images/perspective/${doc.id}`)
-                        .getDownloadURL()
-                        .catch(() => {
-                            console.error("Error: Missing preview image");
-                        })
-                        .then((img) => {
-                            let dat = doc.data();
-                            userVals.push({
-                                name: dat.name,
-                                id: doc.id,
-                                data: dat,
-                                url: img ? img : "/img/no_preview.jpg"
-                            });
-                            num_proj_loaded++;
-                            if(num_proj_loaded === snap.size){
-                                dispatch(syncUserProj(userVals));
-                            }
+            fetch(`${sceneRef}/`, {headers: {"x-access-token": id}}).then((response) =>{
+                if(response.status === 200){
+                    response.json().then((json) =>{
+                        json.forEach(element => {
+                            element.url = `${previewRef}/${element._id}`;
```

Comment:
Hopefully I catch this on the backend too, but can we keep this consistent with our other API calls? So `/apiv1/preview/id/:id`.